src/static/chemiscope.css

```diff
@@ -32,7 +32,7 @@
 }
 
 .chsp-extra-options-btn {
-    margin: auto;
+    margin-right: auto;
```

Comment:
why this change? Both margin should be auto here if we want the button centred in its column

Reply:
I didn't want the button to be centered as then it leaves huge 'gaps' between the button and the rest of the rows.

Reply:
![image](https://user-images.githubusercontent.com/68380659/124128256-e3591c80-da7c-11eb-84ad-90151c868c2c.png)

It seemed neater to have it on the left.

Reply:
but then we have a lot of empty space on the right. Maybe you could increase the width of the select boxes to remove the empty space?

Reply:
![image](https://user-images.githubusercontent.com/68380659/124493171-9bf0ca00-ddb5-11eb-8276-0ffe79ab0acf.png)
good suggestion, it looks better now.